packages/checkbox/src/css/index.js

```diff
@@ -20,13 +20,13 @@ export default {
     height: '16px',
     width: '16px',
     borderRadius: '2px',
-    border: `2px solid ${core.colors.gray02}`,
-    background: core.colors.gray05,
+    border: `2px solid ${core.colorsTextIcon.lowOnDark}`,
+    background: core.colorsBackgroundDark[1],
     color: core.colors.white
   },
   [`.psds-checkbox__square.psds-theme--${themeNames.light}`]: {
-    background: core.colors.white,
-    borderColor: core.colors.gray04,
+    background: core.colorsBackgroundLight[3],
+    borderColor: core.colorsTextIcon.lowOnLight,
```

Comment:
matches figma.  seems odd semantic choice from the figma point of view.  why not colorsBorder?

Reply:
it's figma the square of the checkbox is considered an icon. i agree is is weird but these colors looks better than the border colors. synced up with brian and this is what we came up with